sunpy/util/multimethod.py

```diff
@@ -85,9 +87,8 @@ def add(self, fun, types, override=SILENT):
             raise TypeError
         elif overriden and override == WARN:
             # pylint: disable=W0631
-            warn('Definition ({0}) overrides prior definition ({1}).'.format(_fmt_t(types),
-                                                                             _fmt_t(signature)),
-                 TypeWarning, stacklevel=3)
+            warnings('Definition ({0}) overrides prior definition ({1}).'.format(_fmt_t(types), _fmt_t(signature)),
```

Comment:
Revert this completely. `TypeWarning` is defined in this module and stack level is presumably important here because of the nesting of calls to multimethod. 

Also you have typoed this there is no `.warn` here.

Reply:
done.